charts/rancher-webhook/1.0.2+up0.2.2/Chart.yaml

```diff
@@ -1,8 +1,10 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.22.0-0'
```

Comment:
Actually now that I think about it, we have to be careful about what we put here. Whatever versions of Rancher used this webhook, we have to allow all the supported versions of k8s. 

2.6.0-2.6.1 - 1.18-1.21 - 1.0.0+up0.2.0
2.6.3 - 1.18-1.22 - 1.0.2+up0.2.2
2.6.4 - 1.18-1.23 - 1.0.4+up0.2.5

So that means:
1.0.0 and 1.0.1 need to support < 1.22
1.0.2 needs to support < 1.23 (this file)
1.0.3 and 1.0.4 need to support <1.24 (1.0.3 was introduced in a patch release so being on safe side here)

Can't we just put <1.25 for all of the ones 1.0.2+ though ? Since that's when the next breaking changes are going to be.